traffic_ops/v3-client/server_update_status.go

```diff
@@ -50,8 +50,10 @@ func (to *Session) SetServerQueueUpdate(serverID int, queueUpdate bool) (tc.Serv
 	return resp, reqInf, err
 }
 
-// UpdateServerStatus updates a server's queue status and/or reval status.
+// SetUpdateServerStatuses updates a server's queue status and/or reval status.
 // Either updateStatus or revalStatus may be nil, in which case that status isn't updated (but not both, because that wouldn't do anything).
+//
+// Deprecated: Prefer to use SetUpdateServerStatusTimes
```

Comment:
Same as above RE: GoDoc

Reply:
Removed anyways. No need to specify this in an old, deprecated API.